### ISSUE TITLE
"invalid-function gt-log" on Emacs 31

### ISSUE THREAD

**Issue by nameiwillforget** [lorniu/go-translate#81]
When using go-translate on Emacs 31.0.50, I get the following reply:
```
This test sentence

Google.Detail

invalid-function gt-log

Bing

invalid-function gt-log
```
This was tested using the default use-package config. I tried the same thing on Emacs 30 with the same init and did did not get the same error.

**Comment by lorniu** [comment]
I am using Emacs31 too, but no such problem found. Is the problem still exist? Please provide more using/config details if so.

**Comment by nameiwillforget** [comment]
I don't think so, it might have been resolved through an update, if not I'll reopen.
